dev/test/document.ts

```diff
@@ -170,6 +170,28 @@ describe('serialize document', () => {
     );
   });
 
+  it('serializes large numbers into doubles', () => {
+    const overrides: ApiOverride = {
+      commit: (request, options, callback) => {
+        requestEquals(
+          request,
+          set({
+            document: document('documentId', 'someNumber', {
+              doubleValue: 18014398509481984,
```

Comment:
Can you add a comment to explain what makes this value special? Alternatively, you could also use a more descriptive name than "someNumber".